### DISCUSSION TITLE
New to TailwindCSS - Multiple text styles and various alignment, single row of text.

### DISCUSSION THREAD

**Discussion by marktmcewan** [tailwindlabs/tailwindcss#12591] [Help] [answered]
Hi everyone, I just started using Tailwind this week (after watching a full week of tutorials last week) and loving it so far. I am stuck with the following (image attached from the design)
I am trying to create a line of text as shown: 
"Vegan Flatbread" _2px space then_ "(vgn)" and the price of 9.5 should be right aligned"
All three text have their own text style and should look like this regardless of break points.

I am new to code and I can't figure this one out. I hope you can help. Thanks.

<img width="1683" alt="menu component" src="https://github.com/tailwindlabs/tailwindcss/assets/20793022/4e2e4553-7854-4597-b2ec-98d011bc9d57">

**Comment by wongjn** [comment, marked as answer]
Here are some possible solutions: https://play.tailwindcss.com/iE1UoNYSME

**Reply by marktmcewan** [reply]
Thank you Justin. I didnt know there was something called Tailwind Play either, so thanks for introducing me to that also.

**Comment by alex-krasikau** [comment]
Hey @marktmcewan!

Here's the code for implementing the desired layout:

```js
<div class="flex justify-between items-baseline">
  <div class="flex gap-0.5">
    <p class="text-style-1">Vegan Flatbread</p>
    <p class="text-style-2">(vgn)</p>
  </div>
  <p class="text-style-3">9.5</p>
</div>
```

**Reply by marktmcewan** [reply]
Thanks Alex. Going to go through these code snippets so i can understand what is happening. Thank you for helping so fast also.
